flask_resources/resources.py

```diff
@@ -45,27 +45,27 @@ def __init__(self, config=ResourceConfig):
     # Primary interface
     def search(self, *args, **kwargs):
         """Perform a search over the items."""
-        raise MethodNotAllowed()
+        return {}, 200
```

Comment:
comment: could you open an issue to revisit this behaviour?

Reply:
Sorry! I miight have forgotten to create an issue for others too. Created now: https://github.com/inveniosoftware/flask-resources/issues/54